website/docs/dev_build.md

```diff
@@ -302,120 +403,174 @@ url = "https://distribute.ayon.io/thirdparty/ffmpeg-4.4-windows.zip"
 hash = "dd51ba29d64ee238e7c4c3c7301b19754c3f0ee2e2a729c20a0e2789e72db925"
 # ...
 ```
+
 This defines FFMpeg for Windows. It will be downloaded from specified url, its checksum will
 be validated (it's sha256) and it will be extracted to `/vendor/bin/ffmpeg/windows` (partly taken
 from its section name).
 
 ## Script tools
```

Comment:
@antirotor @iLLiCiTiT you guys should compare the notes on this. I think it's a duplicated section admin/commands docs. @iLLiCiTiT has been removing a bunch of them, so let's keep them in one place only.

Reply:
I think those are commands but here I am documenting the scripts tools (shell helpers). I think what is here still applies.

Reply:
correct. I noticed wrong.